src/index.tsx

```diff
@@ -60,6 +64,8 @@ export {
   PaddedContent,
   Pagination,
   RadioButtonGroup,
+  BaseRadioButtonGroup,
+  RadioButtonGroupContext,
```

Comment:
we shouldn't export the context. the props of the component should be sufficient for consumers.

Reply:
We discussed this on the phone. We need some construct to allow clicks to other content (e.g. a div surrounding a radio button) to trigger the click. I guess another option would be to use a ReactRef and somehow let folks virtually "click" the radio button itself? That doesn't feel quite as nice, tho.

See `it('allows custom components to review and update the value', () => {`

Reply:
HTML allows for this by means of a label on a radio button. I tried the same here, but given the target of our label is just a div as far as the browser is concerned, it doesn't carry through the click.

Does anyone know why we didn't just style a regular ol' radio button? e.g.
https://moderncss.dev/pure-css-custom-styled-radio-buttons/

Sounds like that would allow me to delete yet more code and simplify the scenario?

Reply:
> We discussed this on the phone. We need some construct to allow clicks to other content (e.g. a div surrounding a radio button) to trigger the click. I guess another option would be to use a ReactRef and somehow let folks virtually "click" the radio button itself? That doesn't feel quite as nice, tho.
> 
> See `it('allows custom components to review and update the value', () => {`

An alternative could be to export a component which detects when any of its contents is clicked, which saves the consumer some effort from handling click detection themselves e.g.
```tsx
<RadioButtonGroup ...>
  <RadioButton.CaptureSelection value="...">
    {(renderRadioButton) => (
      <CustomMarkup ...>
      {renderRadioButton()}
    )}
  </RadioButton.Capture>
</RadioButtonGroup>
```

My concerns with exporting a Context are:
a) it's not common for components to expose them
b) we may want to add more data to the context, but not expose that data externally. However, we could do something like have an "internal" context, and expose an "external" context.

Reply:
Yea, I agree with the concern. Yet more motivation to go with a pure-css approach for the base experience. Then Labels just do their job.